frappe/utils/background_jobs.py

```diff
@@ -278,6 +290,7 @@ def start_worker(
 		date_format="%Y-%m-%d %H:%M:%S",
 		log_format="%(asctime)s,%(msecs)03d %(message)s",
 		dequeue_strategy=strategy,
+		with_scheduler=True # ok? https://python-rq.org/docs/scheduling/#safe-importing-of-the-worker-module
```

Comment:
I'm not sure how reliable is use of scheduler for this. We've never used it before because:

> Only a single scheduler can run for a specific queue at any one time. If you run multiple workers with scheduler enabled, only one scheduler will be actively working for a given queue.


Maybe move the scheduler feature to a separate PR where it can be properly tested and reviewed and change the implementation of this feature?

Instead of arbitrary report time, we can send reports in 30/60 minutes intervals. 

I doubt most people need such fine control over time anyway.